app/reset-password/page.tsx

```
@@ -117,46 +138,60 @@ export default function ResetPasswordPage() {
       }
       setError(errorMessage);
     } finally {
+      if (!isMountedRef.current) return;
```

Biome (2.1.2)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)